daft/functions/misc.py

```diff
@@ -99,3 +99,17 @@ def format(f_string: str, *args: Expression | str) -> Expression:
         result = result + expr
 
     return result
+
+
+def file(expr: Expression) -> Expression:
```

Comment:
Sorry for re-raise this PR, may I consult a question about what the difference between `file` function and `file` ExpressionFileNamespace I saw we didn't implement it but we already add a File type, so what use case is better for a function, and what use case for another?

Reply:
@stayrascal, thats a great question. 

Initially we were discussing a `expr.to_file()` method, but ultimately decided to just go with only `file(expr)` as it subjectively _felt_ the most pythonic. `str()`, `list()`, `int()`. If there's a preference for adding a `to_file()` method on expression, feel free to either open up a PR or an issue! 


For more context -- We've gotten a lot of feedback that the namespaces hinder discoverability, So we are actually in the process of removing the expression namespaces all together. There's a bit more context [here](https://github.com/Eventual-Inc/Daft/discussions/4824) about what we plan to do with functions & expressions. Feel free to continue the discussion there!